test_end_to_end/test_s3_direct.yml

```diff
@@ -0,0 +1,11 @@
+addr:
+  # binding address for TCP port that speaks HTTP protocol
+  http: 0.0.0.0:8000
+  # binding address for TCP port that speaks Postgres wire protocol
+  postgres: 0.0.0.0:5432
+tables:
+  - name: "blogs"
+    uri: "s3://test-data/blogs.parquet"
+    option:
+      format: "parquet"
+      use_memory_table: false
```

Comment:
note: this Line is different from `test_end_to_end/test_s3_memory.yml`

Reply:
worth add this as a comment in the config file directly as a documentation.

Reply:
sure, done